GitCommands/Settings/AppSettings.cs

```diff
@@ -1863,6 +1863,12 @@ public static int AutomaticContinuousScrollDelay
             set => SetInt("DiffViewer.AutomaticContinuousScrollDelay", value);
         }
 
+        public static bool LogCaptureCallStacks
+        {
+            get => GetBool("logcapturecallstacks", false);
+            set => SetBool("logcapturecallstacks", value);
```

Comment:
```suggestion
            get => GetBool("Log.CaptureCallStacks", false);
            set => SetBool("Log.CaptureCallStacks", value);
```
?

Reply:
The value is persistent in settings, by changing in Command Log.
Sure, I could make it not persisted but that would be more code.

For name I could change, but lowercase seem more common

Reply:
+1 to @mstv suggestion

Reply:
Changed settings name and force pushed